cfgov/v1/models/base.py

```diff
@@ -217,6 +220,29 @@ def specific_categories_query(block, parent_slug):
         return {search_type: queryset for search_type, queryset in
                 related.items() if queryset}
 
+    def flush(self):
+        if not settings.ENABLE_AKAMAI_CACHE_PURGE:
+            return False
+        auth, fast_purge_url = get_akamai_credentials()
+        page_relative_path = self.url_path.replace("cfgov/", "")
+        headers = {'content-type': 'application/json'}
+        payload = {
+            'action': 'invalidate',
+            'hostname': settings.WAGTAIL_SITE_NAME,
+            'objects': [
+                page_relative_path
+            ]
+        }
+        resp = requests.post(
+            fast_purge_url,
+            headers=headers,
+            data=json.dumps(payload),
+            auth=auth
+        )
+        if resp.status_code == 201:
+            return True
+        return False
```

Comment:
Would it be worth logging anything here about the failure to flush? Is there anything in a non-`201` response that we could log that would help in debugging?